crescent/internal/registry.py

```diff
@@ -294,6 +294,39 @@ async def __post_guild_commands(
                 guild,
             )
 
+    async def purge_commands(
+        self,
+        *guilds: SnowflakeishOr[PartialGuild],
+        skip_global: bool = False,
+        purge_everything: bool = True,
+    ) -> None:
+        """Purge application commands for this Command Handler.
+
+        Args:
+            *guilds: The guilds to purge commands from, if any.
+
+        Kwargs:
+            skip_global:
+                If `True`, skip purging global commands.
+                Defaults to `False`.
+            purge_everything:
+                If `True`, purge all global commands and commands in
+                all `tracked_guilds`. Defaults to `True`.
```

Comment:
Don't think we need to say the default because pdoc shows it anyway. (pdoc my beloved)
```suggestion
        Kwargs:
            skip_global:
                If `True`, skip purging global commands.
            purge_everything:
                If `True`, purge all global commands and commands in
                all `tracked_guilds`. This option takes priority over
                `skip_global`.
```